src/shared/telemetry/defaultTelemetryService.ts

```diff
@@ -99,6 +121,10 @@ export class DefaultTelemetryService implements TelemetryService {
         return this._eventQueue
     }
 
+    public clearRecords(): void {
```

Comment:
Is this only used by tests? Where possible, we try to avoid test-only members in our interfaces. What's the impact on tests if this is removed?

Reply:
It is not completely necessary for testing but is cleaner than some messy hacks to ensure that the service doesn't have any events in its queue when we're testing its behavior.

Reply:
:thumbsup: